data/json/items/obsolete.json

```diff
@@ -517,5 +517,27 @@
         "volume": 1,
         "bashing": 4,
         "to_hit": -2
+    },
+    {
+        "id": "jar_V8",
```

Comment:
I want to turn a "sealed jar of V8" into a "sealed jar" (container size 2) full of "V8" (default charges 1, volume 1).